benchmarks/benchmark/tools/locust-load-inference/sample-terraform.tfvars

```diff
@@ -0,0 +1,24 @@
+credentials_config = {
+  fleet_host = "https://connectgateway.googleapis.com/v1/projects/$PROJECT_NUM/locations/global/gkeMemberships/ai-benchmark"
+}
+
+project_id   = "change-me"
+
+namespace = "benchmark"
+ksa       = "benchmark-ksa"
+
+# Locust service configuration 
+artifact_registry = "us-central1-docker.pkg.dev/$PROJECT_ID/ai-gke-0"
```

Comment:
Does this assume the registry exists or will the module create it?

Reply:
Yes, can you host the pre-created image in a public AR or does the user need to create it each time? Should the README mention how to do that?

Reply:
Locust image build and push is part of the terraform automation, so it's no additional work on the user side. We do not plan to host a public image ourselves for the benchmark at this point in time. With this setup, users have more flexibility for changing the locust user tasks as needed and on terraform apply it will rebuild the image for them.

@achandrasekar possible worth adding a note in the README about the automated build step.

Reply:
Currently the user is expected to create the registry. They can put it in a different registry if they prefer to and the image build and push is automated. But we can simplify this in the future with our hosted image and make custom image an option.